deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -359,28 +359,25 @@ data:
               #     match: after
             paths:
               - /var/log/containers/*${kubernetes.container.id}.log
-      - name: filestream-generic
-        id: hints-container-logs-${kubernetes.hints.container_id}
+      - name: hints-filestream-container-logs
+        id: hints-filestream-container-logs-${kubernetes.hints.container_id}
```

Comment:
Won't be remove?

Reply:
So does this mean that the Makefile is wrong? It looks weird to me why we can't just remove those sections.

Reply:
Well I had introduced them for Kustomize creation. I would say to keep them commented for now in case we want to update the ksutomize templates